---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

### Investigating the initiation of T cell program entry using a new hematopoietic progenitor expansion system

Bone marrow progenitor cells initiate the T cell development program upon encountering T-lineage–promoting environments, primarily through strong Notch signaling. However, studying the earliest responses during this transition has been challenging due to the extremely low frequency and heterogeneity of the relevant progenitor populations. By employing and refining a new hematopoietic progenitor expansion system, we investigated the early chromatin and transcriptomic changes that occur during the initiation of the T cell program. While global chromatin accessibility profiles remain largely unchanged, expanded HSPCs exhibit selective opening or closing of specific regulatory regions in response to Notch signaling, accompanied by immediate gene expression changes at nearby loci. Among the earliest events is the dramatic opening and transcriptional activation of germline T cell receptor β (Trbc1, Trbc2) genes, marking a definitive entry into the T cell lineage. Furthermore,a CRISPR-Cas9 screen using this system uncovered previously unrecognized regulators that regulate the speed of early T cell development, including Hoxa9, Meis1, and Lmo2.
[Shin and Chang et al., 2025 bioRxiv](https://www.biorxiv.org/content/10.1101/2025.04.22.649893v1)

### Runx-induced T-identity program sensitive to Runx dosage, functional co-factors, and gene network    

This paper addressed why Runx factors occupied substantially different sites in the genome in different stages of T cell development as well as in other hematopoietic contexts: quantitative competition of different partner factors for limiting amounts of Runx protein, yet mostly not restricted by local chromatin state. Thus, when this competition was resolved by moderately increasing the availability of Runx factors, early T cell development was dramatically accelerated via additional Runx binding to post-commitment-specific sites precociously even in the pre-commitment stage. Also, a modest increase in Runx levels controlled the activities of other transcription factors, which propagated Runx-mediated gene regulation through an indirect gene network. These findings showed how Runx factors specifically drove the earliest stages of T cell development, how Runx factors confer context-specific gene regulation, and why these effects were sensitive to Runx expression levels in developing T-progenitor cells.
[Shin et al., 2023 Nat Immunol](https://www.nature.com/articles/s41590-023-01585-z)

### Dynamic Runx binding shift and stage-specific gene regulation in early T cell development      

This work focuses on how Runx transcription factors specifically launch the T-identity program in early T cell development, the vastly distinct roles of Runx factors in other hematopoietic lineages.  By disrupting Runx1 and Runx3 simultaneously using the CRISPR-Cas9 system, this study defined the full functional impact of Runx factors, which was previously under-detected due to the functional redundancy. Notably, Runx factors preferentially regulated developmentally dynamic target genes in T-progenitor cells, despite their stable expression kinetics throughout early T-development. In addition, the genes sensitive to loss of Runx factors were substantially different from one stage to the other stage, suggesting highly stage-specific gene regulation. We showed that this switch in regulatory assignments was extensively coordinated with dynamic Runx binding site shifts, as Runx factors were broadly redistributed across the genome, close to their targets, when pro-T cells transited from uncommitted- to the T-committed stages.
[Shin and Hosokawa et al., 2021 PNAS](https://www.pnas.org/doi/10.1073/pnas.2019655118)

### Multi-modular structure of the gene regulatory network for T cell fate specification

T cell gene network models capture the intricate architecture of transcription factor–target gene interactions that govern T cell fate specification. In two recent review papers, we compiled and integrated transcriptomic and genomic datasets to perform a comprehensive global analysis.  This analysis revealed a modular organization within the gene regulatory network, where coherently co-regulated genes cluster into distinct modules that function as discrete units of developmental programming.  Furthermore, these studies demonstrated how transcription factors can collaborate or counteract one another’s activity both within and across modules to shape T cell developmental gene expression programs.
[Shin et al. , 2024 J Exp Med](https://rupress.org/jem/article/221/10/e20230893/276927/Transcriptional-network-dynamics-in-early-T-cell) ;
[Shin and Rothenberg, 2023 Front Immunol](https://www.frontiersin.org/articles/10.3389/fimmu.2023.1108368/full) 


### Effector CD4 T cells with progenitor potential mediate chronic intestinal inflammation

One of our future research directions is to investigate the features of effector CD4 T cells contributing to pathogenesis in chronic inflammatory disorders. This work identified a discrete subset of CD4 T cells with gene expression signatures shared with self-renewing T cells and hematopoietic stem and progenitor cells (HSPCs) during chronic intestinal inflammation.  These progenitor-like effector CD4 T cells could self-renew, escape apoptosis, and continually supply proinflammatory CD4 T cells that sustained and conferred intestinal pathology. This study provided an explanation of how pathogenic CD4 T cells mediated the chronicity of inflammation.  
[Shin et al., 2018 J Exp Med](https://rupress.org/jem/article/215/7/1803/42324/Effector-CD4-T-cells-with-progenitor-potential?searchresult=1)

### The role of mitochondrial OXPHOS in shaping the fate decision between Th17 and regulatory T cells

This paper focused on the effects of mitochondrial oxidative phosphorylation (OXPHOS) controlling pro-inflammatory properties of different subsets of CD4 T cells. It highlighted that ATP-synthase-dependent OXPHOS orchestrated the fate decision between pro-inflammatory Th17 and anti-inflammatory, regulatory T cells. In particular, OXPHOS supported mTOR complex 1 activation in response to the TCR signaling, which was essential for inducing BATF, a key factor for establishing the Th17-chromatin accessibility profile. Consistently, inhibition of OXPHOS during Th17 differentiation prevented Th17-mediated neuroinflammation in the mouse model of multiple sclerosis. 
[Shin et al., 2020 Cell Rep](https://www.cell.com/cell-reports/fulltext/S2211-1247(20)30031-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124720300310%3Fshowall%3Dtrue)

*Complete list:* [Google Scholar](https://scholar.google.com/citations?user=KRA8RWgAAAAJ&hl=en), [NCBI My Bibliography](https://www.ncbi.nlm.nih.gov/myncbi/boyoung.shin.1/bibliography/public/)


